docs/release-notes/2.14.0.md

```diff
@@ -0,0 +1,77 @@
+
+
+## New Features
+
+* [PMM-5765](https://jira.percona.com/browse/PMM-5765): Add external serverless service (no host or address) with new 'pmm-admin add external-serverless' option
+* [PMM-7015](https://jira.percona.com/browse/PMM-7015): Create DB cluster with random password
+* [PMM-6941](https://jira.percona.com/browse/PMM-6941): Integrated Alerting: Ability to delete an 'Alert Rule Template'
+
+
+
+## Improvements
+
+* [PMM-7145](https://jira.percona.com/browse/PMM-7145): 'Push' metrics mode is default when adding services and nodes (All agents collecting data from Services and Nodes will now use PUSH model if not specified explicitly. You will still be able to use '-metrics-mode' flag to use Pull metrics if needed. All previously set up agents will keep their existing mode. To change these you need to remove and re-add them.)
+* [PMM-7282](https://jira.percona.com/browse/PMM-7282): Integrated Alerting: Ability to create rule without channels and filters
+* [PMM-7082](https://jira.percona.com/browse/PMM-7082): Integrated Alerting: Severity levels are color-coded
+* [PMM-7065](https://jira.percona.com/browse/PMM-7065): Integrated Alerting: Show rule details for items in Alert Rules list
+
+
+
+## Bugs Fixed
+
+* [PMM-5137](https://jira.percona.com/browse/PMM-5137): Swagger page redirect isn't working
+* [PMM-7251](https://jira.percona.com/browse/PMM-7251): Integrated Alerting: Error 'Rule with ID "mysql_version" not found' if both Security Threat Tool and Integrated Alerting enabled
+* [PMM-7247](https://jira.percona.com/browse/PMM-7247): Disk size is always 0 for Percona XtraDB cluster
+* [PMM-7178](https://jira.percona.com/browse/PMM-7178): pg_stat_monitor integration is broken: agent stays in 'Waiting' status
+* [PMM-7105](https://jira.percona.com/browse/PMM-7105): Query Analytics: no 'Example' or 'Explain' data for MariaDB
+* [PMM-7239](https://jira.percona.com/browse/PMM-7239): Integrated Alerting: Validate Slack channel names in Notification Channels
+* [PMM-7213](https://jira.percona.com/browse/PMM-7213): MySQL InnoDB Details dashboard: remove color-coding on 'Data Buffer Pool Fit' element
+* [PMM-7167](https://jira.percona.com/browse/PMM-7167): Some panels not visible when using long time intervals (e.g. 30 days)
+* [PMM-7133](https://jira.percona.com/browse/PMM-7133): Incorrect descriptions for data links in dashboards
+* [PMM-6904](https://jira.percona.com/browse/PMM-6904): 'pmm-admin annotate' command crashes for non-generic node types
+* [PMM-6902](https://jira.percona.com/browse/PMM-6902): No query Examples on PostgreSQL 12 with pg_stat_monitor
+* [PMM-6838](https://jira.percona.com/browse/PMM-6838): ProxySQL Instance Summary dashboard: Incorrect "Hostgroup Size" formula
+* [PMM-6490](https://jira.percona.com/browse/PMM-6490): Limit of 100 AWS RDS instances
```

Comment:
wasn't this a community contribution? don't we thank the contributor for that typically?

Reply:
Jirabot automagically accredits community contributions. None of these 4 are community.